doc/tutorial/ranges/range_solution3.cpp

```diff
@@ -1,13 +1,30 @@
 #include <seqan3/core/debug_stream.hpp>
-#include <seqan3/range/views/all.hpp>            // include all of SeqAn's views
+#include <seqan3/argument_parser/all.hpp>       // optional: include the argument_parser
+#include <seqan3/range/views/all.hpp>           // include all of SeqAn's views
 #include <seqan3/std/ranges>                    // include all of the standard library's views
 
+
 int main(int argc, char** argv)
 {
     if (argc == 1)
         return 0;
 
     std::string s{argv[1]};
+    // Optional, use the argument_parser instead:
```

Comment:
Mhmm, I don't understand why we should propose the alternative with the argument parser but only add it as a comment? I would say we always use the argument parser. Otherwise the learners end up with a solution which we would actually not promote.

Reply:
Good point. I changed the solution to use the argument parser.